app/javascript/spec/settings-label-tag-mapping/data.js

```diff
@@ -0,0 +1,34 @@
+export const settingsLabelTagMappingData = () => {
+  const headers = [
+    { header: _('Resource Entity'), key: 'resource_entity' },
```

Comment:
Should these be double-underscore, since this is javascript?  Or does single-underscore have different meaning?

Reply:
single-underscore is used in ruby and doubt-underscore is used in javascript. I got mixed up.